docs/docs/command-line-interface.mdx

```diff
@@ -255,8 +255,8 @@ If you have NLG data for retrieval actions, this will be saved to seperate files
 ```bash
 ls train_test_split
 
-      nlg_test_data.yml     test_data.json
-      nlg_training_data.yml training_data.json
+      nlg_test_data.yml     test_data.yml
+      nlg_training_data.yml training_data.yml
```

Comment:
@akelad after reading the docs page on `rasa data split nlu`, this seemed like the only change required. Did you have something else in mind in your description of https://github.com/RasaHQ/rasa/issues/6588 ?

Reply:
oh, since the response selector training data format changed, i wasn't actually sure that the nlg stuff is still relevant. but @dakshvar22 [PR](https://github.com/RasaHQ/rasa/pull/6591) isn't merged yet, so it may be up to him!

Reply:
It will still be relevant for the time being because response templates for responses still need to be defined outside the regular nlu, stories, rules yml files. See https://github.com/RasaHQ/rasa/issues/6566 cc @degiz

Reply:
wait so `responses:` separately and `responses:` under domain function differently? that seems error prone

Reply:
I'll wait for sasha to answer that since he has been working on it.